tests/dist/src/test/java/org/hawkular/apm/tests/dist/AnalyticsServiceITest.java

```diff
@@ -1159,7 +1160,7 @@ public void testGetCompletionTimeMultiFragment() throws Exception {
         assertNotNull(stats);
         assertEquals(1, stats.size());
 
-        assertEquals(1750, stats.get(0).getAverage());
+        assertEquals(1749999999, stats.get(0).getAverage());
```

Comment:
Looks like source data again - something not converted to microseconds. Should really be 1750000.